tests/scripts/multus/install-cni.yaml

```diff
@@ -0,0 +1,65 @@
+# initially copied from https://github.com/k8snetworkplumbingwg/whereabouts
```

Comment:
do we need this file? I see we are using this once in `setup-multus.sh` script where we can directly use raw version of file to create this yaml? https://raw.githubusercontent.com/k8snetworkplumbingwg/whereabouts/master/hack/cni-install.yml

Reply:
Good idea